zppy/templates/default.ini

```diff
@@ -170,7 +170,8 @@ run_type = string(default="model_vs_obs")
 # "diurnal_cycle" requires `climo_diurnal_subsection`, `climo_diurnal_frequency`, and `dc_obs_climo` to be set.
 # "streamflow" requires `streamflow_obs_ts` to be set.
 # "tc_analysis" requires `tc_obs` to be set.
-sets = string_list(default=list("lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow", "zonal_mean_2d_stratosphere", "tc_analysis"))
+# All available sets = "lat_lon","zonal_mean_xy","zonal_mean_2d","polar","cosp_histogram","meridional_mean_2d","enso_diags","qbo","diurnal_cycle","annual_cycle_zonal_mean","streamflow","zonal_mean_2d_stratosphere","tc_analysis",
+sets = string_list(default=list("lat_lon"))
```

Comment:
@chengzhuzhang @tangq I made `lat_lon` the only default set. Is that what we want? If so, I can go ahead and merge this PR.